### ISSUE TITLE
Option to select where coc installs extensions

### ISSUE THREAD

**Issue by mjarrettjohnson** [neoclide/coc.nvim#393]
Currently, having issues as the root resolution for coc.nvim is installing on a networked drive. It would be great if we could choose where the coc.setting.json and extensions are installed

**Comment by chemzqm** [comment]
What is result of `:echo $HOME` in your vim, is it a networked drive?

**Comment by mjarrettjohnson** [comment]
Yes it was my network drive. Setting the $HOME variable in .vimrc has fixed this and ensured that coc installs its extensions on the right drive

**Comment by chemzqm** [comment]
I've fixed the bug of wrong uri on windows, it should be working now.
